data/data-pipeline/data_pipeline/tests/test_score_calculation.py

```diff
@@ -0,0 +1,286 @@
+from ast import PyCF_TYPE_COMMENTS
+from pathlib import Path
+import pandas as pd
+import pytest
+import data_pipeline.score.field_names as field_names
+import data_pipeline.score.field_names as field_names
+from data_pipeline.utils import get_module_logger
+from data_pipeline.config import settings
+from data_pipeline.score.score_narwhal import ScoreNarwhal
+
+logger = get_module_logger(__name__)
+
+
+@pytest.fixture
+def donut_hole_adjacency_parameters():
+    """Test that donut indicator uses the right threshold"""
```

Comment:
I don't really understand this docstring --- this is not a test?

Reply:
Oh yikes, sorry -- the string should say "parameters to test"...